Fix desktop file.

--- a/src/garglk/gargoyle.desktop
+++ b/src/garglk/gargoyle.desktop
@@ -1,10 +1,9 @@
-#!/usr/bin/env xdg-open
-[Desktop Entry]
-Version=1.0
-Type=Application
-Name=Gargoyle
-GenericName=Interactive Fiction interpreter
-Comment=Interactive Fiction multi-interpreter that supports all major IF formats
-Icon=gargoyle-house.png
-Exec=gargoyle
-Categories=Game;
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Gargoyle
+GenericName=Interactive Fiction interpreter
+Comment=Interactive Fiction multi-interpreter that supports all major IF formats
+Icon=gargoyle-house
+Exec=gargoyle
+Categories=Game;
